new/LIT6843Commentary.xml

```diff
@@ -0,0 +1,90 @@
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6843Commentary" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Commentary on Old Testament books</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="AY"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>This work consists of a Gəʿəz to Gəʿəz commentary on the Pentateuch atributed to 
+                    <persName ref="PRS6978Meherka"/>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Commentary"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="AY" when="2023-06-23">Created entity</change>
+            <change who="AY" when="2023-06-27">Discribe the conetnts  </change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography"><bibl>
+                <ptr target="bm:EMML6"></ptr>
+                <citedRange unit="page">196</citedRange>
+            </bibl>
+                <bibl>
+                    <ptr target="bm:Cowley1988Exeget"></ptr>
+                <citedRange unit="page">114; 118; 432; 438</citedRange>
+                </bibl>
+                <bibl>
+                    <ptr target="bm:Butts2013Reception"></ptr>
+                    <citedRange unit="page">149-151</citedRange>
+                </bibl>
+                </div>
+            <div type="edition">
+                <note>The following edition is based on <ref type="mss" corresp="MDA003"/>.</note>
```

Comment:
As I indicated above, it is better to follow EMML 2101; MDA003 can be included through relations.

Reply:
this comment has not been respected yet.

for the work, please refer to EMML2101 ms. 

MDA003 is secondary, you do not need to mention it in the work record (or add in the witness list)